npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker/version_resolver.rb

```diff
@@ -33,7 +33,7 @@ class VersionResolver
         # peer dependency "react@>=0.14.0 <16.0.0".
```

Comment:
It looks like this code comment documents example error strings... does it need updating as well? I'm unclear if this is a new error string or a slight tweak on an existing one...

Reply:
Yes, let me update it!

Reply:
Updated!

Reply:
Oh that is so much better, I hadn't even realized the `\` was intended to signal multi-line break. 😁 

❤️